docs/spec.md

```diff
@@ -593,23 +593,34 @@ Performs element-wise addition of two tensors `lhs` and `rhs` and produces a
 * For integers: integer addition.
 * For floats: `addition` from IEEE-754.
 * For complex numbers: complex addition.
+* For quantized types:
+  * `float_result = (lhs - zero_point(lhs)) * scale(lhs) +
+    (rhs - zero_point(rhs)) * scale(rhs)`.
+  * `rounded_result = round_nearest_even(float_result / scale(result))`.
```

Comment:
Floor implementation can be varied per hardware. For example, some cases prefer floor(x+ 0.5) instead of round_nearest_even.

Can we consider not specifying the round method here?

Reply:
If this explanation describes just one possible way to calculate the given case, it should be fine.

Reply:
Hi @abattery 

>> Round implementations can be varied per hardware.
I agree. The current variant of round follows from https://mlir.llvm.org/docs/Quantization/#relation and it is one one possible way.